content-scripts/inject/module.js

```diff
@@ -302,7 +302,7 @@ if (isScratchGui || isProject) {
   }
 }
 
-if (location.pathname === "/discuss/3/topic/add/") {
+if (location.pathname.includes("topic") || location.pathname.includes("discuss/settings/")) {
```

Comment:
It would make sense to use Regex here.

Reply:
Will do in a bit, i'm just really bad at regex so it might take awhile

Reply:
You might like https://github.com/ziishaned/learn-regex

Reply:
alright, done
> You might like [ziishaned/learn-regex](https://github.com/ziishaned/learn-regex)

thank you for that!